### PR TITLE
Allow Pin to Reset and Restart

### PR DESCRIPTION
**Pull request by michaelwehrley**
Allow Pin to Reset and Restart

* Add a input to allow for a a physician to restart the app.

[Finishes: #93562236]

### CODE REVIEW

File: src/js/controllers/PagesController.js [hunk 1]
```diff
@@ -52,9 +55,17 @@
         this.currentCategory.setPhysician(physician);
       }
     };
+
+    this.restart = function(event) {
+      event.preventDefault();
+
+      if (this.pin === '1111') {
```
Comment (ericcf) [inline src/js/controllers/PagesController.js]: I'd suggest not hard coding this value. Make it part of the build process, or make it configurable the first time the app is run would be my suggestion.

Comment (michaelwehrley) [inline src/js/controllers/PagesController.js]: Thanks. I saw this in later versions and made the change. :-)

Comment (michaelwehrley) [inline src/js/controllers/PagesController.js]: @ericcf see: https://github.com/cbitstech/ed_navigator_app/blob/986f4b9936940acc76f298b3ed8396e732b169bb/src/js/controllers/PagesController.js#L87

Comment (ericcf) [inline src/js/controllers/PagesController.js]: Hmm...it's still hard coded in the linked version.

Comment (michaelwehrley) [inline src/js/controllers/PagesController.js]: The link works for me, but how about this:
https://github.com/cbitstech/ed_navigator_app/blob/msw_93560130_10_second_chapter_view/src/js/controllers/PagesController.js#L87
https://github.com/cbitstech/ed_navigator_app/blob/msw_93560130_10_second_chapter_view/src/js/controllers/PagesController.js#L37
https://github.com/cbitstech/ed_navigator_app/blob/msw_93560130_10_second_chapter_view/src/js/controllers/PagesController.js#L54

**Comment by michaelwehrley** [comment]
@ericcf I made the adjustment.  Thanks!

**Comment by ericcf** [comment]
Thanks! :icecream:

**Pull request merged**